src/main/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSpark.java

```diff
@@ -229,73 +229,70 @@ static SGAAssemblyResult runSGAModulesInSerial(final String sgaPathString,
         // collect runtime information along the way, if user requests
         final List<SGAModule.RuntimeInfo> runtimeInfo = new ArrayList<>();
 
-        final List<String> moduleArgs = new ArrayList<>(20); // 2*10 is not a magic number, just that no SGA modules have more than 10 sensibly tunable parameters
+        final Map<String, String> moduleArgsAndValues = new LinkedHashMap<>(10); // 10 is some arbitrary number of make enough capacity
```

Comment:
Can you just adjust this comment a little? Maybe "10 is the maximum number of arguments to any SGA modules"